gantry/commands/logs.py

```diff
@@ -56,7 +75,7 @@ def logs(
             else:
                 raise ConfigurationError(f"Invalid task name '{task_name}'")
         else:
-            job = beaker.workload.get_latest_job(wl)
+            j = _get_job(beaker, wl, tasks[0], run=run)
```

Comment:
I don't really understand that change, but trust that this won't break other features.

Reply:
The `beaker.workload.get_latest_job` method defaults to the first task, so this should be equivalent.